config/management/src/config.rs

```diff
@@ -54,6 +54,7 @@ use structopt::StructOpt;
 
 /// Config for aptos management tools
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(deny_unknown_fields)]
```

Comment:
Why?  Also, we can't do this now that many people have a config that is incompatible, unless we want to annoy thousands of node operators.

Reply:
That's fine, we're an evolving project and bad fields are bad, they are confusing, and they are magical. I was not happy when we removed this and am very eager to see this put back into place.